tests/e2e/helpers/common.ts

```diff
@@ -188,4 +191,34 @@ export class Common {
     static async getPageUrl(): Promise<string> {
         return (await ClientFunction(() => window.location.href))();
     }
+
+    /**
+      * Create Zip archive from folder
+      * @param folderPath Path to folder to archive
+      * @param zipName Zip archive name
+     */
+    static async createZipFromFolder(folderPath: string, zipName: string): Promise<void> {
+        const sourceDir = path.join(__dirname, folderPath);
+        const zipFilePath = path.join(__dirname, zipName);
+        const output = fs.createWriteStream(zipFilePath);
+        const archive = archiver('zip', { zlib: { level: 9 } });
+
+        // Add the contents of the directory to the zip archive
+        archive.directory(sourceDir, false);
+
+        // Finalize the archive and write it to disk
+        archive.finalize();
+        await new Promise((resolve) => {
+            output.on('close', resolve);
+            archive.pipe(output);
+        });
```

Comment:
could you please clarify why the promise is need here? it looks like it won't work as async code

Reply:
thanks, looks like it can be simplified
fixed